app/src/main/java/com/framgia/englishconversation/screen/dialog/OptionPostViewModel.java

```diff
@@ -0,0 +1,47 @@
+package com.framgia.englishconversation.screen.dialog;
+
+import android.databinding.BaseObservable;
+import android.util.Log;
```

Comment:
ERROR: Unused import: android.util.Log. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports